package.json

```diff
@@ -61,6 +61,6 @@
     "number-to-words": "1.2.2",
     "require-dir": "0.3.0",
     "snyk": "1.14.3",
-    "student-debt-calc": "2.3.0"
+    "student-debt-calc": "^2.4.1"
```

Comment:
Please remove the carat to pin this dependency, then re-shrinkwrap the packages. :-)